salesagility/SuiteCRM
# Fixed #994 - Address Translation

## Description



When navigating to http://{PATH_TO_INSTANCE}/index.php/ the trailing slash causes the page styling to disappear along with increased load times.

![issue-994](https://user-images.githubusercontent.com/26431166/28968059-174fa8a0-7916-11e7-9218-d0affcf16b86.png)

This change updates install_utills to add functionality to .htaccess on install/upgrade that will remove the trailing slash when attempting to access /index.php/

## Motivation and Context

Issue reference: #994

## How To Test This

1. Install or upgrade SuiteCRM (Alternatively run rebuild htaccess in admin->repair).
2. Navigate to http://{PATH_TO_INSTANCE}/index.php/
3. Check that the page is styled with the correct theme

## Types of changes


### Final checklist
